src/main/java/dev/lsegal/jenkins/codebuilder/CodeBuilderCloud.java

```diff
@@ -59,6 +59,7 @@ public class CodeBuilderCloud extends Cloud {
   private static final String DEFAULT_JNLP_IMAGE = "lsegal/jnlp-docker-agent:alpine";
   private static final int DEFAULT_AGENT_TIMEOUT = 120;
   private static final String DEFAULT_COMPUTE_TYPE = "BUILD_GENERAL1_SMALL";

```

Comment:
I configured cloud agent on jenkins with BUILD_GENREAL1_MEDIUM for WINDOWS 9 platform with image mcr.microsoft.com/dotnet/framework/sdk:4.8, but when job triggers it pickup the Default value which is BUILD_GENERAL1_SMALL, so had to change this value in config.xml manually and restart the jenkins

Second issue, I faced is about PrivilegedMode, in  awscodebuild project by default PrivilegedMode is set as false but still seeing below error . 
[CodeBuilder]: Exception while starting build: PrivilegedMode is not supported for WINDOWS_SERVER_2019_CONTAINER projects (Service: AWSCodeBuild; Status Code: 400; Error Code: InvalidInputException; Request ID: cf069de4-9b5e-479f-8875-cc86a6f87dbe; Proxy: null)